src/l10n/nl.js

```diff
@@ -0,0 +1,353 @@
+            'Export Passwords'                    : 'Wachtwoorden exporteren',
+            'Export Folders'                      : 'Mappen exporteren',
+            'Export Tags'                         : 'Tags exporteren',
+            'FolderLabel'                         : 'Maplabel',
+            'TagLabels'                           : 'Taglabels',
+            'ParentLabel'                         : 'Bovenliggende map',
+            'FolderId'                            : 'Map Id',
+            'TagIds'                              : 'Tag Ids',
+            'ParentId'                            : 'Bovenliggende map Id',
+            'Empty'                               : 'Leeg',
+            'Add Header Line'                     : 'Kopregel toevoegen',
+            'Export passwords shared with me'     : 'Exporteer wachtwoorden die met mij gedeeld zijn',
+            'Don\'t edit passwords shared with me': 'Geen wachtwoorden veranderen die met mij gedeeld zijn',
+            'Run Export'                          : 'Export uitvoeren',
+            'Export'                              : 'Exporteren',
+            'Waiting...'                          : 'Loopt...',
+            'Export error'                        : 'Export Fout',
+            'Download {format}'                   : 'Download {format}',
+            'Nothing to export'                   : 'Niks te exporteren',
+            'There is no data to export'          : 'Er is geen data voorhanden om te exporteren',
+            'Restore or import'                   : 'Restore of importeren',
+            'Passwords CSV'                       : 'Wachtwoorden CSV',
+            'Folder CSV'                          : 'Mappen CSV',
+            'Custom CSV'                          : 'Overige CSV',
+            'Select File'                         : 'Selecteer bestand',
+            'Select Options'                      : 'Selecteer opties',
+            'Conflict handling'                   : 'Conflict afhandeling',
+            'Skip if same revision'               : 'Overslaan wanneer zelfde revisie',
+            'Skip always'                         : 'Altijd overslaan',
+            'Overwrite existing'                  : 'Bestaande overschrijven',
+            'Merge with existing'                 : 'Samenvoegen met bestaande',
+            'Create new entry'                    : 'Nieuw record aanmaken',
+            'For encrypted backups'               : 'Verplicht bij versleutelde backups',
+            'CSV Options'                         : 'CSV Opties',
+            'Database'                            : 'Database',
+            'Quote Character'                     : 'Aanhalingsteken',
+            'Escape Character'                    : 'Escape karakter',
+            'Field Delimiter'                     : 'Veldbegrenzer',
+            'Quote'                               : 'Hochkomma',

```

Comment:
Is "Hochkomma" correct as translation for a double quote?

Reply:
The translation in Dutch for "Quote" and "Double Quote"  is the same. "Hochkomma" is 
German, changed it to " Aanhalingsteken".